.gitignore

```diff
@@ -29,4 +29,47 @@ _ReSharper*/
 [Tt]est[Rr]esult*
 Commencement.SQL/Commencement.SQL.dbmdl
 *.dbmdl
-*.docstates
\ No newline at end of file
+*.docstates
+Backup/PTF.sln
```

Comment:
Assuming you really do want to ignore/keep all of this, you can use wildcards instead of listing every file.  Example, PTF.CLR/Backup* should hide most of these in one line